libraries/RC_Channel/RC_Channels.cpp

```diff
@@ -231,6 +237,21 @@ bool RC_Channels::flight_mode_channel_conflicts_with_rc_option() const
     return (RC_Channel::aux_func_t)chan->option.get() != RC_Channel::AUX_FUNC::DO_NOTHING;
 }
 
+// clear overrides by radio input
+void RC_Channels::clear_overrides_by_rc()
+{
+    if ((rc().clear_overrides_by_roll() && channel(AP::rcmap()->roll() - 1)->radio_input_changed()) ||
+        (rc().clear_overrides_by_pitch() && channel(AP::rcmap()->pitch() - 1)->radio_input_changed()) ||
+        (rc().clear_overrides_by_throttle() && channel(AP::rcmap()->throttle() - 1)->radio_input_changed()) ||
+        (rc().clear_overrides_by_yaw() && channel(AP::rcmap()->yaw() - 1)->radio_input_changed())) {
+        set_gcs_overrides_enabled(false);
+        gcs().send_text(MAV_SEVERITY_NOTICE, "Clear overrides by pilot input");
```

Comment:
This comment should probably include "RC" somewhere.  So perhaps "RC overrides cleared by pilot input"

Reply:
fixed